py_analyserver/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM socrata/py_nltk
+
+RUN pip install Flask>=0.10.1 \
```

Comment:
We use the [default python-pip that comes with Trusty](http://packages.ubuntu.com/trusty/python/python-pip), i.e. 1.5.4.